x-pack/plugins/apm/server/lib/services/annotations/get_derived_service_annotations.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { isNumber } from 'lodash';
+import { Annotation, AnnotationType } from '../../../../common/annotations';
+import { SetupTimeRange, Setup } from '../../helpers/setup_request';
+import { ESFilter } from '../../../../typings/elasticsearch';
+import { rangeFilter } from '../../helpers/range_filter';
+import {
+  PROCESSOR_EVENT,
+  SERVICE_NAME,
+  SERVICE_ENVIRONMENT,
+  SERVICE_VERSION
+} from '../../../../common/elasticsearch_fieldnames';
+
+export async function getDerivedServiceAnnotations({
```

Comment:
Perhaps add a comment here that explains what a derived annotation is (compared to a normal annotation)

Reply:
done